spec/views/smoke_tests_spec.rb

```diff
@@ -43,4 +43,13 @@
       expect{ render template: "pages/edit" }.not_to raise_error
     end
   end
+
+  describe "index" do
+    it 'renders without error' do
+      3.times { create(:page) }
```

Comment:
here we can just use `build` instead of `create` so it doesn't have to hit the database (kept those unit tests fast and all)